calculation/gmhazard_calc/gmhazard_calc/dbs/IMDB.py

```diff
@@ -330,7 +330,7 @@ def rupture_names(self) -> Union[None, np.ndarray]:
 
     @check_open
     def im_data(
-        self, station: str, im: Optional[Union[List[IM], IM]] = None, inter_intra: bool = False,
+        self, station: str, im: Optional[Union[List[IM], IM]] = None, incl_with_between_sigma: bool = False,
```

Comment:
one nitpicking, if we decide to change the name from `inter_intra` to something more meaningful,
Shall we use `incl_within_between_sigma`? 
It feels slightly odd to me seeing with and between in the variable and based on the docstring, it says `between-event` and `within-event` instead of `with-event`, so when the variable got longer, might as well add extra 2 characters? :thinking: 
However, this is my personal opinion, so up to you to keep or change it :+1: